tests/integrations/mcs/resourcemanager/resource_manager_test.go

```diff
@@ -789,7 +789,7 @@ func (suite *resourceManagerClientTestSuite) TestBasicResourceGroupCURD() {
 		}
 		// Create Resource Group
 		resp, err := cli.AddResourceGroup(suite.ctx, group)
-		checkErr(err, tcase.addSuccess)
+		checkErr(err, true)
 		if tcase.addSuccess {
```

Comment:
Maybe it's better to change `addSuccess` name to meet now the test AddResourceGroup behavior will actually succeed.

Reply:
done